### ISSUE TITLE
Subsurface scattering support (SSSS)

### ISSUE THREAD

**Issue by straaljager** [ospray/ospray#269]
Hi,

I checked out the new Principled shader introduced in Ospray 1.6, but couldn't find a subsurface parameter as in Disney's original PBR shader implementation (https://github.com/wdas/brdf/blob/master/src/brdfs/disney.brdf) 

Is there some way Ospray can do SSS? 

Thanks

**Comment by carsonbrownlee** [comment]
@johguenther may give a better answer, but as far as I know we do not yet compute any subsurface scattering in our shaders.  however, people asking for it is a good way of getting new features in ospray down the line:)

**Comment by johguenther** [comment]
Agreed, having SSS in OSPRay would be cool. But please don't expect it very soon, many other features also wait to be included...

**Comment by straaljager** [comment]
Thanks for your replies. Good to know it's on the radar.

**Comment by chegarty3** [comment]
I know this is an old post, but I would add my interest in subsurface scattering.  Even just a backscattering component in combination with transmission color & transmission depth would be valuable.
